build_projects/dotnet-cli-build/DotNetTest.cs

```diff
@@ -50,5 +52,15 @@ private string GetXml()
 
             return null;
         }
+
+        private string GetNoBuild()
+        {
+            if (NoBuild)
+            {
+                return "--no-build";
+            }
+
+            return null;
```

Comment:
`null` is the convention that was set for this use case in the first check-in. @eerhardt likely has details on `why`